src/main/groovy/com/marklogic/gradle/MarkLogicPlugin.groovy

```diff
@@ -210,6 +211,7 @@ class MarkLogicPlugin implements Plugin<Project> {
 
 		String roxyGroup = "ml-gradle Roxy";
 		project.task("mlRoxyCopyProperties", type: RoxyCopyPropertiesTask, group: roxyGroup, description: "Copy Roxy properties to gradle.properties file")
+		project.task("mlCopyRoxyFiles", type: CopyRoxyFilesTask, group: roxyGroup, description: "Copy roxy files")
```

Comment:
@divino For consistency, can you name this "mlRoxyCopyFiles" and "RoxyCopyFilesTask"? I'd like to have all of the Roxy-related tasks start with "mlRoxy".